samples/GameMenu/GameMenu.cpp

```diff
@@ -1,4 +1,4 @@
-/***********************************************************************
+﻿/***********************************************************************
```

Comment:
what is this change about?

Reply:
File encoding changed, utf-8 no BOM to utf-8 with BOM, as the special character "❚" in line 711 can't decode well in some code page such as 936.
utf-8 with BOM will make VS happy.

Reply:
alright